lib/src/Uncertainty/Distribution/openturns/ParametrizedDistribution.hxx

```diff
@@ -146,6 +146,21 @@ public:
   /** Get the standard representative in the parametric family, associated with the standard moments */
   Distribution getStandardRepresentative() const override;
 
+  /** Compute the PDF of Xi | X1, ..., Xi-1. x = Xi, y = (X1,...,Xi-1) */
+  using DistributionImplementation::computeConditionalPDF;
```

Comment:
CodeCov states that these methods are not tested:

![image](https://github.com/user-attachments/assets/e8a87687-d798-4a1d-bdc0-81fde7e9f4c7)

Reply:
done